.github/workflows/platformDeploy.yml

```diff
@@ -28,6 +28,25 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.OS_BOTIFY_TOKEN }}
 
+  # Note: we're updating the checklist before running the deploys and assuming that it will succeed on at least one platform
```

Comment:
I think this makes sense.

Let's say they all fail... the failure message appears on the PR only. But this already happens today and I assume the deployer will already have eyes on that PR -- given that they are waiting on the successful CP